pkg/util/cluster/cluster.go

```diff
@@ -50,6 +50,11 @@ type Cluster struct {
 	roleassignments   authorization.RoleAssignmentsClient
 }
 
+const (
+	FirstPartyClientIDProduction  = "f1dd0a37-89c6-4e07-bcd1-ffd3d43d8875"
+	FirstPartyClientIDIntegration = "8aa2578d-0948-477f-b157-02d3fce19fa0"
```

Comment:
Unexport these.  In pkg/env/int.go we're using "71cfb175-ea3a-444e-8c03-b119b2752ce4" - which one's right?  Ideally we'd consolidate all of these into single consts in a separate package but it needn't be this PR.

Reply:
In secrets has 8aa... so which one is right? O_o